src/components/VSelect/VSelect-combobox.spec.js

```diff
@@ -77,6 +77,31 @@ test('VSelect - combobox', ({ mount }) => {
     expect('Unable to locate target [data-app]').toHaveBeenTipped()
   })
 
+  it('should not use search input if an option is selected from the menu', async () => {
+    const item = { value: 123, text: 'Foo' }
+    const wrapper = mount(VSelect, {
+      propsData: {
+        combobox: true,
+        items: [item]
+      }
+    })
+
+    const event = jest.fn()
+    wrapper.vm.$on('input', event)
+
+    wrapper.setData({ isActive: true })
+    await wrapper.vm.$nextTick()
+
+    wrapper.vm.selectItem(item)
+    await wrapper.vm.$nextTick()
+
+    wrapper.setData({ isActive: false })
+    await wrapper.vm.$nextTick()
+
+    expect(event).toBeCalledWith(item)
+    expect('Unable to locate target [data-app]').toHaveBeenTipped()
```

Comment:
Merge in the most recent updates from dev. @jacekkarczmarczyk added better error/warning messages and this test will fail.

Reply:
I just merged in the most recent (at time of writing) `dev` branch ([0d75322e](https://github.com/rokkie/vuetify/commit/0d75322ebedc87d58f503c8d70cbb7d0ed592db0)) but as far as i can tell the test passes (at least on my machine..). Am i doing it wrong or did things change in the meantime?

Reply:
My change add the "[Vuetify]" prefix to the error message, `toHaveBeenTipped` function checks if the console messages **includes** the text from `expect` so it still passes. No need imho to change anything here